db/models/models.js

```diff
@@ -70,3 +88,22 @@ exports.insertComment = (body, article_id) => {
 		})
   }
 
+  
+
+
+
+  exports.updateArticleById = (article_id, inc_votes) => {
+    if (!inc_votes || typeof inc_votes !== "number") {
```

Comment:
You don't need to manually check these, inc_votes is set to INT NOT NULL in the database, which means that if you try to update with the wrong data or no data - PSQL will return an error for you. You could then catch this error, send it to next, and then deal with it there. 

There's nothing wrong with your implementation, but we can handle it better. Upto you whether you want to refactor!